### PR TITLE
Feature: add properties and fix Serialization

### PR DESCRIPTION
**Pull request by rainer-helbing**
added properties ValueDetails and ValueType 
Serialization of `Feature`  fixed: _FDESCRIPTION_ => _FDESCR_, added `lang` attribute like in other places

### CODE REVIEW

File: OpenTrans.net/Feature.cs [hunk 1]
```diff
@@ -62,5 +67,10 @@ public class Feature
         /// Definition of the feature
         /// </summary>
         public string Template { get; set; }
+
+        /// <summary>
+        /// Information on the valid value range
+        /// </summary>
+        public string ValueType { get; set; }
```
Comment (stephanstapel) [inline OpenTrans.net/Feature.cs]: how about making this an enum? According to the docs, there are only three valid values

**Comment by rainer-helbing** [comment]
I prefer not to. I don't think that this property is often used and if the file contains something else, we are in trouble again.
________________________________
Von: Stephan ***@***.***>
Gesendet: Samstag, 14. Oktober 2023 18:46
An: stephanstapel/OpenTrans.net ***@***.***>
Cc: Rainer Helbing ***@***.***>; Author ***@***.***>
Betreff: Re: [stephanstapel/OpenTrans.net] Feature: add properties and fix Serialization (PR #44)


@stephanstapel commented on this pull request.

________________________________

In OpenTrans.net/Feature.cs<https://github.com/stephanstapel/OpenTrans.net/pull/44#discussion_r1359479768>:

> @@ -62,5 +67,10 @@ public class Feature
         /// Definition of the feature
         /// </summary>
         public string Template { get; set; }
+
+        /// <summary>
+        /// Information on the valid value range
+        /// </summary>
+        public string ValueType { get; set; }


how about making this an enum? According to the docs, there are only three valid values

**Comment by stephanstapel** [comment]
The standard says there is no option for custom value types though

**Comment by rainer-helbing** [comment]
Changed ValueType to enum

**Pull request merged**